pom.xml

```diff
@@ -450,9 +449,15 @@
                                             <skip>true</skip>
                                             <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-debezium:${camel-quarkus-test-list.version}</artifact>
                                         </test>
-                                        <test><!-- https://github.com/apache/camel-quarkus/issues/4058 -->
+                                        <test>
+                                            <!-- Workaround flaky Kudu test: https://github.com/apache/camel-quarkus/issues/4259 -->
+                                            <skip>true</skip>
+                                            <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-kudu:${camel-quarkus-test-list.version}</artifact>
+                                        </test>
+                                        <test>
+                                            <!-- Workaround flaky XML test : https://github.com/apache/camel-quarkus/issues/4258 -->
                                             <skip>true</skip>
-                                            <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-cxf-soap:${camel-quarkus-test-list.version}</artifact>
+                                            <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-xml:${camel-quarkus-test-list.version}</artifact>
                                         </test>
```

Comment:
This can be removed given that we skip only the failing tests in test-pom.xsl: https://github.com/quarkusio/quarkus-platform/pull/692/files#diff-83ce56bfbc63c6c743ed24689a3e000a2a06e2857c874c85c82b9ca19149c73aR135-R141

Reply:
I pushed an update as I need this merged sooner rather than later :).